frontend/app/components/mapbox/filter-features.js

```diff
@@ -0,0 +1,31 @@
+import Component from '@ember/component';
+import { computed } from '@ember-decorators/object';
+
+export default class MapboxFilterFeaturesComponent extends Component {
```

Comment:
`Mapbox::FeatureFilterer` ? for consistency ?

Reply:
👍 I like it. Will change.